### PR TITLE
Write ODK version to makefile

### PR DESCRIPTION
**Pull request by matentzn**
Long standing wish of mine to actually see with which ODK a makefile was created!

### CODE REVIEW

**Comment by matentzn** [comment]
Checking seems to have same issue as https://github.com/INCATools/ontology-development-kit/pull/262

If someone could look into this:
https://github.com/INCATools/ontology-development-kit/pull/262#issuecomment-517702814

File: odk/odk.py [hunk 1]
```diff
@@ -472,7 +472,7 @@ def generate(self, input : str) -> str:
         """
         with open(input) as file_:
             template = Template(file_.read())
-            return template.render( project = self.context.project)
+            return template.render( project = self.context.project, env=os.environ)
```
[the remark this answers is not in the document]
Comment (matentzn) [reply odk/odk.py]: @althonos can judge this perhaps as well :P

[the remark this answers is not in the document]
Comment (althonos) [reply odk/odk.py]: Do you really need all environment variables to be exposed, or just a few ?

[the remark this answers is not in the document]
Comment (matentzn) [reply odk/odk.py]: Only one! Can I somehow transfer individual environment variables?

[the remark this answers is not in the document]
Comment (althonos) [reply odk/odk.py]: @matentzn : `env = {"MYVAR": os.getenv("MYVAR")}` if you don't want to edit anything else, but best would be `myvar=os.getenv("MYVAR")` but you'll need to change some stuff in the template as well.

[the remark this answers is not in the document]
Comment (matentzn) [reply odk/odk.py]: Peeeerfect. Thanks @althonos Did that now. Will merge when tests are passing!

**Pull request merged**